### PR TITLE
Tweak model building log messages

### LINKED ISSUE #11249: Relationship from 'User' to 'Customer' not configured because there are multiple properties on one entity type
Since updating to 2.1.0-preview1, I noticed these messages popping at startup.
These are apparently due to multiple foreign keys on one table referencing the same table.

For example : 

Customer -> What user created the customer (references User)
Customer -> What user modified the customer (references User)

There are no errors or exceptions thrown but the message spam considerably slows down startup.
My application with 170+ keys takes a full minute to start, and it worked fine in 2.0.1.

The models and context are straight from Scaffold-DbContext with no alteration.

### Steps to reproduce

Here is a repro for the issue.

```c#
using Microsoft.EntityFrameworkCore;
using Microsoft.Extensions.Logging;
using System;
using System.Collections.Generic;

namespace MultipleRelationshipsBug
{
    class Program
    {
        static void Main(string[] args)
        {
            using (var context = new MyContext())
            {
                context.Database.EnsureCreated();
            }

            Console.Read();
        }
    }

    public partial class MyContext : DbContext
    {
        public virtual DbSet<Customer> Customer { get; set; }
        public virtual DbSet<User> User { get; set; }

        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        {
            if (!optionsBuilder.IsConfigured)
            {
                optionsBuilder.UseSqlServer(@"Server=.;Database=MultipleRelationshipsBug;Trusted_Connection=True;MultipleActiveResultSets=true")
                                .EnableSensitiveDataLogging()
                                .UseLoggerFactory(new LoggerFactory().AddConsole(LogLevel.Debug));
            }
        }

        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            modelBuilder.Entity<Customer>(entity =>
            {
                entity.Property(e => e.Id).HasColumnName("id");

                entity.Property(e => e.CreatedByFk).HasColumnName("createdBy_fk");

                entity.Property(e => e.ModifiedByFk).HasColumnName("modifiedBy_fk");

                entity.HasOne(d => d.CreatedByFkNavigation)
                    .WithMany(p => p.CustomerCreatedByFkNavigation)
                    .HasForeignKey(d => d.CreatedByFk)
                    .OnDelete(DeleteBehavior.ClientSetNull)
                    .HasConstraintName("FK_Customer_UserCreated");

                entity.HasOne(d => d.ModifiedByFkNavigation)
                    .WithMany(p => p.CustomerModifiedByFkNavigation)
                    .HasForeignKey(d => d.ModifiedByFk)
                    .OnDelete(DeleteBehavior.ClientSetNull)
                    .HasConstraintName("FK_Customer_UserModified");
            });

            modelBuilder.Entity<User>(entity =>
            {
                entity.Property(e => e.Id).HasColumnName("id");
            });
        }
    }

    public partial class User
    {
        public User()
        {
            CustomerCreatedByFkNavigation = new HashSet<Customer>();
            CustomerModifiedByFkNavigation = new HashSet<Customer>();
        }

        public int Id { get; set; }

        public ICollection<Customer> CustomerCreatedByFkNavigation { get; set; }
        public ICollection<Customer> CustomerModifiedByFkNavigation { get; set; }
    }

    public partial class Customer
    {
        public int Id { get; set; }
        public int CreatedByFk { get; set; }
        public int ModifiedByFk { get; set; }

        public User CreatedByFkNavigation { get; set; }
        public User ModifiedByFkNavigation { get; set; }
    }
}
```

My .csproj
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.0-preview1-final" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.1.0-preview1-final" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.0-preview1-final" />
    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.1.0-preview1-final" />
  </ItemGroup>

  <ItemGroup>
    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.1.0-preview1-final" />
  </ItemGroup>

</Project>
```

### Further technical details
EF Core version: 2.1.0-preview1-final
Database Provider: Microsoft.EntityFrameworkCore.SqlServer
Operating system: Windows 7 x64
IDE: Visual Studio 2017 15.6.0

### PR DESCRIPTION
**Pull request by AndriySvyryd**
Fixes #11249

### CODE REVIEW

File: src/EFCore/Properties/CoreStrings.resx [hunk 1]
```diff
@@ -1023,11 +1023,11 @@
     <comment>Warning CoreEventId.ConflictingShadowForeignKeysWarning string string string</comment>
   </data>
   <data name="LogMultipleNavigationProperties" xml:space="preserve">
-    <value>Relationship from '{firstEntityType}' to '{secondEntityType}' not configured because there are multiple properties on one entity type {navigationProperties} that could be matched with the properties on the other entity type {inverseNavigations}.</value>
+    <value>No relationship from '{firstEntityType}' to '{secondEntityType}' has been configured by convention because there are multiple properties on one entity type {navigationProperties} that could be matched with the properties on the other entity type {inverseNavigations}.</value>
```
Comment (ajcvickers) [inline src/EFCore/Properties/CoreStrings.resx]: I think we may want to be even more clear than this that these don't necessarily indicate any errors/mistakes in your code. But not sure how to word this. @divega Any thoughts?

Comment (divega) [reply src/EFCore/Properties/CoreStrings.resx]: I agree that is exactly what we decided in triage: we wanted to make that clearer that this was a just a signal that one of the preconditions of the convention was not met but that it wasn't even a warning. I am just not sure what to do to the text to better convey that.

And on the other hand I am starting to see that the stream of messages you'll see coming from model building is going to be by nature very advanced and there are diminishing returns in trying to make them look super simple.

Comment (AndriySvyryd) [reply src/EFCore/Properties/CoreStrings.resx]: We can add: `This message can be disregarded if explicit configuration has been specified.`

Comment (ajcvickers) [reply src/EFCore/Properties/CoreStrings.resx]: Sounds good.

**Pull request merged**